tools/tests/toast/test.js

```diff
@@ -0,0 +1,179 @@
+const toastTemplate = `
```

Comment:
Make a function that accepts the parameters as objects so you can test different scenarios. If top is top, or bottom is bottom, etc. See rangeslider tests

Reply:
Fixed